src/core/js/logging.js

```diff
@@ -0,0 +1,78 @@
+define(['core/js/adapt'], function(Adapt) {
```

Comment:
Ignore the referenced/copied standard, it hasn't been formalized yet. It should say "spaces 4" indentation, no tabs, etc.